### PR TITLE
Grafana config in terraform

### PR DESCRIPTION
**Pull request by mattkaem**
* added outputs needed for grafana config
* added inputs in software/main.tf needed for grafana config

### CODE REVIEW

File: environment-template/software/main.tf [hunk 1]
```diff
@@ -8,9 +8,12 @@ module "software" {
   sql_user                            = data.terraform_remote_state.infrastructure.outputs.sql_user
   sql_db_pw                           = data.terraform_remote_state.infrastructure.outputs.sql_db_pw
   sql_ip                              = data.terraform_remote_state.infrastructure.outputs.sql_ip
-  sql_database                        = data.terraform_remote_state.infrastructure.outputs.sql_database
+  sql_hono_database                   = data.terraform_remote_state.infrastructure.outputs.sql_hono_database
+  sql_grafana_database                = data.terraform_remote_state.infrastructure.outputs.sql_grafana_database
   service_name_communication          = data.terraform_remote_state.infrastructure.outputs.service_name_communication
   device_communication_static_ip_name = data.terraform_remote_state.infrastructure.outputs.device_communication_static_ip_name
+  grafana_static_ip_name              = try(data.terraform_remote_state.infrastructure.outputs.grafana_static_ip_name[0], "")
+  grafana_expose_externally           = try(data.terraform_remote_state.infrastructure.outputs.grafana_expose_externally[0], false)
```
Comment (julian-sotec) [inline environment-template/software/main.tf:16]: Can you please cxplain why the grafana_expose_externally property is an array?

File: environment-template/software/main.tf [hunk 2]
```diff
@@ -8,9 +8,12 @@ module "software" {
   sql_user                            = data.terraform_remote_state.infrastructure.outputs.sql_user
   sql_db_pw                           = data.terraform_remote_state.infrastructure.outputs.sql_db_pw
   sql_ip                              = data.terraform_remote_state.infrastructure.outputs.sql_ip
-  sql_database                        = data.terraform_remote_state.infrastructure.outputs.sql_database
+  sql_hono_database                   = data.terraform_remote_state.infrastructure.outputs.sql_hono_database
+  sql_grafana_database                = data.terraform_remote_state.infrastructure.outputs.sql_grafana_database
   service_name_communication          = data.terraform_remote_state.infrastructure.outputs.service_name_communication
   device_communication_static_ip_name = data.terraform_remote_state.infrastructure.outputs.device_communication_static_ip_name
+  grafana_static_ip_name              = try(data.terraform_remote_state.infrastructure.outputs.grafana_static_ip_name[0], "")
```
Comment (julian-sotec) [inline environment-template/software/main.tf:15]: Can you please cxplain why the grafana_static_ip_name property is an array?

File: environment-template/software/main.tf (hunk 1 above)
Comment (mattkaem) [reply environment-template/software/main.tf]: I changed this to not be an array and access the data without try

File: environment-template/software/main.tf (hunk 2 above)
Comment (mattkaem) [reply environment-template/software/main.tf:15]: The grafana static IP resource is created conditionally by setting count to 0 or 1 and this leads to the output being an array.

File: environment-template/software/main.tf (hunk 1 above)
Comment (julian-sotec) [reply environment-template/software/main.tf]: understood!

File: environment-template/software/main.tf (hunk 2 above)
Comment (julian-sotec) [reply environment-template/software/main.tf:15]: Okay, that make sense!

**Review by julian-sotec — APPROVED** [review]
Looks good to me

**Pull request merged**